website/templates/include/comment_pane_template.mako

```diff
@@ -0,0 +1,49 @@
+<div class="scripted comment-pane">
+
+    <div class="cp-handle-div cp-handle pull-right pointer hidden-xs" data-bind="click:removeCount" data-toggle="tooltip" data-placement="bottom" title="Discussion Pane">
+        <span data-bind="if: unreadComments() !== 0">
+            <span data-bind="text: displayCount" class="badge unread-comments-count"></span>
+        </span>
+        <i class="fa fa-comments-o fa-2x comment-handle-icon"></i>
+    </div>
+    <div class="cp-bar"></div>
+
+
+    <div class="comments cp-sidebar">
+        <div class="cp-sidebar-content">
+            <button type="button" class="close visible-xs text-smaller" data-bind="click: togglePane">
+                <i class="fa fa-times"></i>
+            </button>
+            <h4>
+                <span data-bind="if: page() == 'node' ">${node['title']} Discussion</span>
```

Comment:
Is there a reason to use mako here instead of Knockout? Does using mako prevent a FOUC?